.github/workflows/apidoc.yaml

```diff
@@ -23,7 +23,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        apiGroup: [ 'observability-api', 'public-api', 'management-api', 'control-api' ]
+        apiGroup: [ 'public-api', 'management-api', 'control-api' ]
```

Comment:
i think this was explicitly added in https://github.com/eclipse-edc/Connector/pull/3859. Do we not want to publish the Observability API anymore?